### DISCUSSION TITLE
Хочу смотреть мультики на ДБГ

### DISCUSSION THREAD

**Discussion by KillerBunny031** [octothorp-team/public-dobrograd#841] [Предложения]
Давайте добавим рабочий телевизр, на котором можно будет смотреть разные каналы. Как сейчас с радио.

Это придаст сразу несколько плюсов для разных категорий игроков. 

Гражданские
---
Придаст еще одну причину купить квартиру и находиться в ней. 

Криминал
---
Еще одна вещь, которую можно будет украсть. 

Новые организации
---
Появятся новое пространство для организаций. Например, как зараждающийся новострой канал "News Totay"

**Comment by doxzter** [comment]
Это нереализуемо без поддержки хромиума (бета версия гмода)

**Comment by Unknow123456789** [comment]
Кто-то просто поставит телики на спауне и включит там порнуху.
